joomla/joomla-cms
# [4.1] Fire change events consistently on the media Field

Pull Request for Issue #37640 .

### Summary of Changes

- Call the Custom Element function setValue instead of manually updating the value

### Testing Instructions

- Edit an article and go to the images tab
- Open the inspector and inspect the intro image input (the one left to the select button bellow the image)
- type in the browser's console `$0.addEventListener('change', (e) => console.log(e))`
- Select an image, check that a change event was recorded in the console
- Clear the field, check the recorded event
- Select another image and check that the event was recorded again

### Actual result BEFORE applying this Pull Request

No event on image selection

### Expected result AFTER applying this Pull Request



### Documentation Changes Required

No this is a bug
